custom/logistics/management/commands/clear_verified_numbers.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management import BaseCommand
+from corehq.apps.sms.mixin import VerifiedNumber
+from custom.ewsghana.models import EWSGhanaConfig
+from custom.ilsgateway.models import ILSGatewayConfig
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        logistics_domains = ILSGatewayConfig.get_all_enabled_domains() + EWSGhanaConfig.get_all_enabled_domains()
+        for domain in logistics_domains:
```

Comment:
This makes me nervous - can't we just make this command take a list of domains as args to do this for?